lib/display.rb

```diff
@@ -0,0 +1,33 @@
+class Display
+
+  def initialize(console = Console.new, message)
+    @console = console
+    @message = message
+  end
+
+  public
+
+  def present_board(grid)
+    position_in_array = 0
+    while position_in_array < grid.size
+      @console.present(grid.slice(position_in_array, size_of_grid(grid)).join(" "))
+      position_in_array += size_of_grid(grid)
+    end
+  end
+  
+  def size_of_grid(grid)
+    Math.sqrt(grid.size)
+  end
+
+  def welcome
+    @console.present(@message.welcome)
+  end
+
+  def choose_cell
+    @console.present(@message.prompt_for_cell)
+  end
+
+  def receive_cell_choice
```

Comment:
This method doesn't seem specific to receiving a cell choice. I could be receiving any input to convert to an int.